domain/build.gradle

```diff
@@ -1,15 +1,12 @@
 
 dependencies {
     annotationProcessor 'org.projectlombok:lombok:1.18.12'
-
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.10.3'
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.10.3'
-
-
-
+    compile 'org.springframework.boot:spring-boot-starter-web:2.0.1.RELEASE'
```

Comment:
We should be consistent when adding new dependencies and follow the `group`-`name`-`version` declaration style